<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTray</class>
 <widget class="QDialog" name="addTray">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="timeSpinbox">
     <property name="suffix">
      <string> ms</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Start well</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="startSpinbox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>End well</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="endSpinbox"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="confirm">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirm</sender>
   <signal>accepted()</signal>
   <receiver>addTray</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm</sender>
   <signal>rejected()</signal>
   <receiver>addTray</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
